exe/qless-config

```diff
@@ -0,0 +1,45 @@
+#! /usr/bin/env ruby
+
+require 'json'
+require 'qless'
+require 'thor'
+
+class Configurator < Thor
+
+  class_option :redis, :default => 'redis://localhost:6379/0'
```

Comment:
I'll change it, though this repo probably needs some TLC before turning on `rubocop`. Sadly, this isn't the time to give it the care and feeding it needs.